app/writer.py

```diff
@@ -65,6 +69,9 @@ def set_times(e, times):
     e.add('dtstart', times[0])
     e.add('dtend', times[1])
 
+def remove_events_on_menu(cal, menu_start):
+    """ Removes any events from `cal` that have been "rescraped" """
```

Comment:
I'll take your word for it.  I really should have put this comment on line 55 though because that is where the concern stems from.